docs/fsharp-core-notes.md

```diff
@@ -1,126 +1,124 @@
 
-    <PackageReference Update="FSharp.Core" Version="4.7.2" />
+### Do *not* bundle FSharp.Core directly with a library 
 
-In C# projects use:
+Do _not_ include a copy of FSharp.Core with your library or package, such in the `lib` folder of a package. If you do this, you will create havoc for users of your library.
 
-    <PackageReference Include="FSharp.Core" Version="4.7.2" />
+The decision about which `FSharp.Core` a library binds to is up to the application hosting of the library.
 
-If you make your FSharp.Core dependency explicit, you will have to explicitly upgrade your FSharp.Core reference in order to use
-new F# language or library features should those features depend on a particular minimal FSharp.Core version.
+## Guidance for everyone else
 
-### Libraries should target lower versions of FSharp.Core
+If you're not authoring packages for distribution, you have a lot less to worry about.
 
-F# ecosystem libraries should generally target the *earliest, most portable* profile of FSharp.Core feasible, within reason.
+If you are distributing library code across a private organization as if it were a NuGet package, please see the above guidance, as it likely still applies. Otherwise, the below guidance applies.
 
-If your library is part of an ecosystem, it can be helpful to target the _earliest, most widespread language version_ 
-and the _earliest_ and _most supported_ profiles of the .NET Framework feasible.
+### Application authors don't have to explicitly reference FSharp.Core
 
-The version you choose should be based on the minimum F# language version you want to support. The minimum FSharp.Core version for each language version is listed below:
+In general, applications can always just use the latest FSharp.Core bundled in the SDK they are built with.
 
-|Minimum F# language version|Minimum FSharp.Core version|
-|------------------------------|------------------------------|
-|F# 4.1|4.3.4|
-|F# 4.5|4.5.2|
-|F# 4.6|4.6.2|
-|F# 4.7|4.7.2|
-|F# 5.0|5.0.0|
+### C# projects referencing F# projects may need to pin FSharp.Core
+
+You can reference an F# project just fine without needing to be explicit about an FSharp.Core reference when using non-legacy projects. References flow transitively, so even if you need to use types directly from FSharp.Core (which you probably shouldn't do anyways) it will pick up the right types from the right assembly.
 
-A good choice for libraries is to target `netstandard2.0` and FSharp.Core 4.7.2.
+If you do have an explicit FSharp.Core reference in your C# project that you **need**, you should pin your FSharp.Core reference across your entire codebase. Being in a mixed pinned/non-pinned world is difficult to keep straight over a long period of time.
 
-    <PackageReference Update="FSharp.Core" Version="4.7.2" />
+## Guidance for legacy projects, compilers, and tools
```

Comment:
Define "legacy" and give an example of the head of a typical legacy project file.  Don't assume people know the "SDK" v. "non-SDK" langauge

Reply:
I will instead link out to here. Although I would disagree; at this point we've managed to make it fairly clear to the broader .NET population what this means